File: python/7 kyu/285.py
# https://www.codewars.com/kata/564e7fc20f0b53eb02000106/train/python

# Passed

def consonant_count(s):
    vowels = {'a', 'e', 'i', 'o', 'u'}
    consonants_in_string = [letter for letter in s if letter.isalpha() and letter.lower() not in vowels]
    consonants_count = len(consonants_in_string)
    return consonants_count

output = consonant_count('h^$&^#$&^elLo world')
print(output)

File: nodejs/6 kyu/285.js
// https://www.codewars.com/kata/52840d2b27e9c932ff0016ae/train/javascript

// Passed

function locate(arr, value) {
  for (const elem of arr) {
    if (Array.isArray(elem)) {
      let valueFound = locate(elem, value);
      if (valueFound) {
        return true;
      }
    } else {
      if (elem === value) {
        return true;
      }
    }
  }

  return false;
}

const output = locate(["a", "b", ["c", "d", ["e"]]], "d");
console.log(output);